pyspecProcScripts/convert_to_power.py

```diff
@@ -108,12 +134,15 @@ def convert_to_power(filename, exptype, s, fl=None):
         # mean_power_vs_time = prscr.dBm2power(mean_power_vs_time)
         mean_power_vs_time.set_units("W")
         # }}}
-    fl.plot(
-        mean_power_vs_time, "o"
-    )  # this  should be a *single* o at the center of each power step.
-    #    Its y value should be the avaerage power for that step, and its
-    #    error bars should give the standard deviation of the power over
-    #    the step
+    if fl:
+        fl.plot(
+            mean_power_vs_time, "o"
+        )  # this  should be a *single* o at the center of each power step.
+        #    Its y value should be the avaerage power for that step, and its
+        #    error bars should give the standard deviation of the power over
+        #    the step
     # }}}
-    s["indirect"].data = mean_power_vs_time.data
+    s.setaxis("indirect", mean_power_vs_time.data).set_error(
+        "indirect", mean_power_vs_time.get_error()
+    ).set_units("indirect", "W")
```

Comment:
the `s['indirect']` is a numpy array.  It has a `data` attribute, but I think that's not what you're going for here. 

btw, what I'm doing in the setaxis call is the same as `s['indirect'] = mean_power_vs_time.data` I just like stringing the calls together.